lib/dor/workflow/client/workflow_routes.rb

```diff
@@ -157,14 +157,35 @@ def workflow_xml(repo, druid, workflow)
         # ==
         # The method does an HTTP PUT to the URL defined in `Dor::WF_URI`.
         #
-        #     PUT "/dor/objects/pid:123/workflows/GoogleScannedWF/convert"
+        #     PUT "/objects/pid:123/workflows/GoogleScannedWF/convert"
         #     <process name=\"convert\" status=\"error\" />"
-        def update_workflow_error_status(repo, druid, workflow, process, error_msg, opts = {})
-          opts = { error_text: nil }.merge!(opts)
-          xml = create_process_xml({ name: process, status: 'error', errorMessage: error_msg }.merge!(opts))
-          requestor.request "#{repo}/objects/#{druid}/workflows/#{workflow}/#{process}", 'put', xml, content_type: 'application/xml'
+        def update_workflow_error_status(_repo, druid, workflow, process, error_msg, opts = {})
+          update_error_status(druid: druid, workflow: workflow, process: process, error_msg: error_msg, error_text: opts[:error_text])
           true
         end
+        deprecation_deprecate update_workflow_error_status: 'use update_error_status instead.'
+
+        # Updates the status of one step in a workflow to error.
+        # Returns true on success.  Caller must handle any exceptions
+        #
+        # @param [String] druid The id of the object
+        # @param [String] workflow The name of the workflow
+        # @param [String] error_msg The error message.  Ideally, this is a brief message describing the error
+        # @param [Hash] opts optional values for the workflow step
+        # @option opts [String] :error_text A slot to hold more information about the error, like a full stacktrace
+        # @return [Boolean] always true
+        #
+        # Http Call
+        # ==
+        # The method does an HTTP PUT to the URL defined in `Dor::WF_URI`.
+        #
+        #     PUT "/objects/pid:123/workflows/GoogleScannedWF/convert"
+        #     <process name=\"convert\" status=\"error\" />"
+        def update_error_status(druid:, workflow:, process:, error_msg:, error_text: nil)
```

Comment:
I don't see `process` or `error_msg` in the YARD doc above. Worth adding them?

Reply:
Yes. I'll add that.